docs/DE0006.md

```diff
@@ -18,38 +18,67 @@ T:System.Collections.CaseInsensitiveHashCodeProvider
-| `Comparer`                        | `Comparer<T>`                                              |
-| `CaseInsensitiveComparer`         | `StringComparer.OrdinalIgnoreCase`                         |
-| `CaseInsensitiveHashCodeProvider` | `StringComparer.OrdinalIgnoreCase`                         |
+The following table shows how the non-generic collection types can be
+replaced by their generic counterparts from the [`System.Collections.Generic`](generic) or
+[`System.Collections.ObjectModel`](objectmodel) namespaces:
+
+| Type                                                                 | Replacement                                                                                   |
+|----------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
+| [`ArrayList`](arraylist)                                             | [`List<T>`](list)                                                                             |
+| [`CaseInsensitiveComparer`](caseinsensitivecomparer)                 | [`StringComparer.OrdinalIgnoreCase`](ordinalignorecase)                                       |
+| [`CaseInsensitiveHashCodeProvider`](caseinsensitivehashcodeprovider) | [`StringComparer.OrdinalIgnoreCase`](ordinalignorecase)                                       |
+| [`CollectionBase`](collectionbase)                                   | [`Collection<T>`](collection-1)                                                               |
+| [`Comparer`](comparer)                                               | [`Comparer<T>`](comparer-1)                                                                   |
+| [`DictionaryBase`](dictionarybase)                                   | [`Dictionary<TKey, TValue>`](dictionary) or [`KeyedCollection<TKey, TItem>`](keyedcollection) |
+| [`DictionaryEntry`](dictionaryentry)                                 | [`KeyValuePair<TKey, TValue>`](keyvaluepair)                                                  |
+| [`Hashtable`](hashtable)                                             | [`Dictionary<TKey, TValue>`](dictionary)                                                      |
+| [`Queue`](queue)                                                     | [`Queue<T>`](queue-1)                                                                         |
+| [`ReadOnlyCollectionBase`](readonlycollectionbase)                   | [`ReadOnlyCollection<T>`](readonlycollection)                                                 |
+| [`SortedList`](sortedlist)                                           | [`SortedList<TKey, TValue>`](sortedlist-2)                                                    |
+| [`Stack`](stack)                                                     | [`Stack<T>`](stack-1)                                                                         |
```

Comment:
Brackets!
![image](https://user-images.githubusercontent.com/5169960/36275083-b3d43d3c-123e-11e8-9cb6-23a79896308d.png)

Reply:
(Also, typos, but hey, I created the GIF, so....)

Reply:
LOL! You can tell I'm not used to this kind of link. Argh....